pkg/configuration/digger_config.go

```diff
@@ -44,11 +44,12 @@ type GenerateProjectsConfig struct {
 }
 
 type Project struct {
-	Name       string `yaml:"name"`
-	Dir        string `yaml:"dir"`
-	Workspace  string `yaml:"workspace"`
-	Terragrunt bool   `yaml:"terragrunt"`
-	Workflow   string `yaml:"workflow"`
+	Name             string   `yaml:"name"`
+	Dir              string   `yaml:"dir"`
+	Workspace        string   `yaml:"workspace"`
+	Terragrunt       bool     `yaml:"terragrunt"`
+	Workflow         string   `yaml:"workflow"`
+	DependentModules []string `yaml:"dependent_modules,omitempty"`
```

Comment:
I think dependent here would be "this modules dependent on our project", but it's the other way. Should be something like "dependency_modules"